src/main/java/org/opentripplanner/standalone/config/routerconfig/VectorTileConfig.java

```diff
@@ -71,7 +80,22 @@ public static VectorTileConfig mapVectorTilesParameters(NodeAdapter node, String
           is expected to be handled by a proxy.
           """
         )
-        .asString(DEFAULT.basePath)
+        .asString(DEFAULT.basePath),
+      root
+        .of("attribution")
+        .since(V2_5)
+        .summary("Custom attribution to be returned in `tilejson.json`")
+        .description(
+          """
+          By default the `attribution` property in `tilejson.json` is computed from the names and 
+          URLs of the feed publishers.
+          If the OTP deployment contains many fields, this can become very unwieldy. 
+          
+          This configuration parameter allows you to set the `attribution` to any string you wish,
```

Comment:
"any string you wish" => "any string you wish, including HTML tags"

Reply:
e55c2c91d234de98f15dde60401823bf051f23ff